roles/kubernetes-apps/ansible/templates/nodelocaldns-daemonset.yml.j2

```diff
@@ -42,9 +43,7 @@ spec:
         - -localip
         - {{ nodelocaldns_ip }}
         - -conf
-        - /etc/coredns/Corefile
-        - -upstreamsvc
```

Comment:
why removing the upstreamsvc?

Reply:
I explain this in the commit message, but this is a no-op in the Kubespray configuration, since we us Ansible to template the Corefile. The sole purpose of upstreamsvc is to replace `__PILLAR__CLUSTER__DNS__` during templating from `Corefile.base` with the IP of the upstream service.